libusb/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+cmake_minimum_required(VERSION 3.6.3 FATAL_ERROR)
```

Comment:
This should be in the very root of the project, not in a subdirectory.

Reply:
unfortunately, we have different minimum version requirements for different backends
my intention is to keep lowest minimum version possible, and still use modern CMake with minimum compromises